src/software/ai/evaluation/detect_successful_kick_test.cpp

```diff
@@ -0,0 +1,45 @@
+#include "software/ai/evaluation/detect_successful_kick.h"
+
+#include <gtest/gtest.h>
+
+
+TEST(DetectSuccessfulKickTest, ball_over_speed_threshold_and_no_direction_difference)
+{
+    Ball ball({0, 0}, {5, 5}, Timestamp::fromSeconds(0));
+
+
```

Comment:
Nit: Remove one of the lines to keep it consistent with rest of the tests

Reply:
removed